machines/openIdBle/request.ts

```diff
@@ -380,9 +383,18 @@ export const requestMachine =
                   },
                 },
               },
+              on: {
+                STORE_ERROR: {
+                  target: '#request.reviewing.savingFailed',
+                },
+              },
             },
             accepted: {
-              entry: ['sendVcReceived', 'logReceived'],
+              entry: [
+                'sendVcReceived',
+                'setReceiveLogTypeRegular',
```

Comment:
This can be moved to actions section of VC_RECEIVED event and avoid this change. Apparently actions section also supports array.

Reply:
There are multiple log types which are set based on accepted or rejected. As rejected flow needs a different type of log type we can't move it to VC_RECEIVED section.

Reply:
Since on VC_RECEIVED event, we are always moving to accepting state, it should be fine in my opinion. 
But it's fine to ignore this.